handler_test.go

```diff
@@ -123,6 +123,36 @@ func TestUpdateStar(t *testing.T) {
 	}
 }
 
+func TestUpdateStarDoesNotDuplicate(t *testing.T) {
+	var m mock.Mock
+	currentUser := bson.NewObjectId()
+	m.On("One", &item{}).Return(nil).Run(func(args mock.Arguments) {
+		*args.Get(0).(*item) = item{ID: "stable/wordpress", StargazersIDs: []bson.ObjectId{currentUser}}
+	})
+	dbSession = testutil.NewMockSession(&m)
+	oldGetCurrentUserID := getCurrentUserID
+	getCurrentUserID = func(_ *http.Request) (bson.ObjectId, error) { return currentUser, nil }
+	defer func() { getCurrentUserID = oldGetCurrentUserID }()
+	tests := []struct {
+		name        string
+		requestBody string
+		wantCode    int
+	}{
+		{"noop", `{"id": "stable/wordpress", "has_starred": true}`, http.StatusOK},
```

Comment:
since there is only ever going to be one test case for this, we probably don't need the table driven test